src/features/defi/providers/yearn/components/YearnManager/Withdraw/YearnWithdraw.tsx

```diff
@@ -435,7 +435,14 @@ export const YearnWithdraw = ({ api }: YearnWithdrawProps) => {
     >
       <YearnRouteSteps routes={routes} />
       <Flex flexDir='column' width='full' minWidth='400px'>
-        {withdrawRoute && <DefiActionButtons />}
+        {withdrawRoute && (
+          <DefiActionButtons
+            expired={
+              !!state.vault.metadata.depositsDisabled ||
```

Comment:
Do we need to coerce `state.vault.metadata.depositsDisabled`? It's already of type `boolean | undefined`.

Reply:
fixed